lib/index.js

```diff
@@ -8,7 +8,7 @@ const ttf2woff2 = require('ttf2woff2')
 const Fontmin = require('fontmin')
 const RawSource = require('webpack-sources').RawSource
 
-const FONT_REGEX = /\.(eot|ttf|svg|woff|woff2)$/
+const FONT_REGEX = /\.(eot|ttf|svg|woff|woff2)(\?.+)*$/
```

Comment:
```suggestion
const FONT_REGEX = /\.(eot|ttf|svg|woff|woff2)(\?.+)?$/
```

I wouldn't expect this to repeat, correct?

Reply:
you're right, fixed.